pkg/usage/usage.go

```diff
@@ -67,7 +67,7 @@ func (db *DB) GetUsageByDateRange(startDate time.Time, endDate time.Time) ([]*Us
 	usageStartDate := time.Date(startDate.Year(), startDate.Month(), startDate.Day(), 0, 0, 0, 0, time.UTC)
 	usageEndDate := time.Date(endDate.Year(), endDate.Month(), endDate.Day(), 23, 59, 59, 0, time.UTC)
 
-	if usageEndDate.Sub(usageStartDate) < 0 {
+	if usageEndDate.Sub(usageStartDate) <= 0 {
```

Comment:
I think this should be < 0 and fix error message from "should be before" to "should be on or before"